### DISCUSSION TITLE
[A-L] (2023/24) Foglio 3 - Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#184] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-10-22 204457](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/2f68fe5b-c82b-4a0d-99b0-c103afbdbcd7)
    ($^1$)In conclusione, le soluzioni di (1) si ottengono sommando ad una soluzione particolare di (1) tutte le soluzioni dell’equazione omogenea associata.

**Comment by Elia-Belli** [comment, marked as answer]
- **(1)** $897$ e $4403$ sono coprimi se $(897,4403)=1$
Usando l'algoritmo di Euclide:
$4403=897\cdot 4 +815$
$897=815\cdot1+82$
$815=82\cdot9+ 77$
$82=77\cdot 1 +5$
$77=5\cdot 15 + 2$
$5=2\cdot 2+ 1$
$2=1\cdot 2 + 0$
$\Rightarrow (897,4403)=1$
- **(2)** Cerchiamo un'identità di Bezout:
$815 = a-b\cdot 4=(1,0)+(0,-4)=(1,-4)$
$82=b-815\cdot 1=(0,1)+(1,-4)(-1)=(-1,5)$
$77=815-82\cdot 9=(1,-4)+(-1,5)(-9)=(10,-49)$
$5=82-77\cdot 1=(-1,5)+(10,-49)(-1)=(-11,54)$
$2=77-5\cdot 15=(10,-49)+(-11,54)(-15)=(175,-859)$
$1= 5-2\cdot 2=(-11,54)+(175,-859)(-2)=(-361,1772)$
$\Rightarrow (\tilde{x},\tilde{y})=(1772,-361)$
- **(3)** Vogliamo verificare che $897(\tilde{x}+x_0)+4403(\tilde{y}+y_0)=1$
Per associatività $897\tilde{x}+897x_0+4403\tilde{y}+4403y_0=1$
Per commutatività $(897x_0+4403y_0)+897\tilde{x}+4403\tilde{y}=1$
Infine per ipotesi $0+897\tilde{x}+4403\tilde{y}=1$ è verificata.

  Sia $(x',y'): 897x'+4403y'=1\Rightarrow 897x'+4403y'=897\tilde{x}+4403\tilde{y}$
  $\Rightarrow 897x'+4403y'-897\tilde{x}-4403\tilde{y}=0\Rightarrow 897(x'-\tilde{x})+4403(y'-\tilde{y})=0$, quindi $(x'-\tilde{x},y'-\tilde{y})$ è soluzione dell'eq omogenea associata, allora esiste $(x_0,y_0)=(x'-\tilde{x},y'-\tilde{y})$ tale che:
$(x_0,y_0)=(x',y')-(\tilde{x},\tilde{y})\Rightarrow (x',y')=(\tilde{x},\tilde{y})+(x_0,y_0)$ è verificata.

- **(4)** [Soluzione di @CiottoloMaggico](https://github.com/sapienzastudentsnetwork/algebra2324/discussions/184#discussioncomment-7863455)

**Comment by CiottoloMaggico** [comment]
Analogo a @Elia-Belli tranne punto 4:
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/9c790b57-83cb-4ded-aa3a-7cdb7a3737bc)

**Comment by AglaiaNorza** [comment]
Soluzione ad opera della tutor [Elisa Scandiuzzi](https://github.com/sapienzastudentsnetwork/algebra/blob/main/scandiuzzi/README.md), lezione del 23 ottobre 2024 (`appunti_24_10_23.pdf`):

![1](https://github.com/user-attachments/assets/6a9870fb-ad07-4a67-9cad-9e7089c80d5b)
![2](https://github.com/user-attachments/assets/1f29ca10-c5fc-4d1f-a8ff-e3e1d6348281)
![3](https://github.com/user-attachments/assets/f089dc9d-737e-4f22-9628-8ebf17506f30)
